frontend/src/RequestOptions/task-requests.js

```diff
@@ -0,0 +1,56 @@
+/**Task-related options  */
+
+/*  preferred update method:  the frontend will send new task item with updated changes 
+expecting the backend to find the task and replace it with new one
+*/
+function updateTasks (taskItem) {
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({type:'update_task', taskItem})
+    };
+    fetch('https://localhost:8080/endpoint.php', requestOptions);
+    
+}
+
+/* update method as requested by backend */
+function updateTask (taskItem) {
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({type:'update_task', task_id: taskItem.id, completed: taskItem.completed})
```

Comment:
... is also applied here (and everywhere else)?

Reply:
See commented lines about the update method. There's one that is preferred (to just send the whole object) and one that complies with the backend documentation of what is expected. Let me know what you'd like. Do you want me to send the backend the whole object for the rest of the requests as well?

Reply:
Yes, assume that #38 has already been fixed and the backend is expecting the format you wrote above for `update_task`. I know this won't work until #38 is fixed, which creates a bit of a pickle. I will create a pull request to fix #38 after #36 is merged.

Reply:
Alternatively, you can change the format `update_task` to the old format, but in any case just make sure their is consistency

Reply:
`update_task` is according to the old format. I'm confused now, do you want the new format or old?

Reply:
Sorry for the confusion. Pick one format for the body of all the POST requests, new or old format doesn't matter as long as there is consistency. Old format would allow you to test your code, but you will need to change it to the new format eventually

Reply:
We can also talk during scrum

Reply:
Agreed, that is why I kept both versions. These are just functions, so I can call the one according to the situation. Is that okay?